xtask/codegen/html.ungram

```diff
@@ -147,3 +153,57 @@ HtmlString = value: 'html_string_literal'
 HtmlTagName = value: 'html_literal'
 HtmlAttributeName = value: 'html_literal'
 HtmlContent = value: 'html_literal'
+
+// ==================================
+// Astro-specific syntax nodes
+// ==================================
+
+// Astro expressions: {expression}
+HtmlAstroExpression =
+	l_curly: '{'
```

Comment:
For these, I think it would be better to prefix them with just `Astro` instead of `HtmlAstro`. This would be more consistent with how js and ts nodes are named.